intTests/test_comp_bisim/feedback.saw

```diff
@@ -0,0 +1,94 @@
+enable_experimental;
+import "cryptol/Feedback.cry";
+import "cryptol/Spec.cry";
```

Comment:
Either we should change the names of these `.cry` files to start with an uppercase letter, or we should make this change:

```suggestion
import "cryptol/feedback.cry";
import "cryptol/spec.cry";
```